apm-sniffer/apm-sdk-plugin/mysql-6.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/jdbc/mysql/v6/define/LoadBalancedConnectionProxyInstrumentation.java

```diff
@@ -52,4 +51,9 @@
     @Override protected ClassMatch enhanceClass() {
         return byName(INTERCEPT_CLASS);
     }
+
+    @Override
+    protected String[] witnessClasses() {
```

Comment:
I advise that this method should remove. because this method had implemented in its parent class